mm/MemOps.c

```
@@ -276,442 +56,27 @@ void* FastMemcpy(void* restrict dest, const void* restrict src, uint64_t size) {
     uint8_t* d = dest;
     const uint8_t* s = src;
 
-    // Handle overlap cases
     if (d == s || !d || !s) return dest;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)